apps/miranum-console/package.json

```diff
@@ -103,7 +103,7 @@
 			  {
 				"submenu": "deploy.submenu",
 				"group": "1_modification",
-				"when": "resourceExtname == .bpmn || resourceExtname == .form || resourceExtname == .dmn"
+				"when": "resourceExtname == .bpmn || resourceExtname == .form || resourceExtname == .dmn || resourceExtname == .json"
```

Comment:
should also check if it's in the "config" folder, but I can't get "resourceDirname" (https://code.visualstudio.com/api/references/when-clause-contexts) to work the way I want